public/src/main/java/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -40,7 +46,11 @@ data class EntitlementInfo constructor(
     val unsubscribeDetectedAt: Date?,
     val billingIssueDetectedAt: Date?,
     val ownershipType: OwnershipType,
-) : Parcelable {
+    val jsonObject: JSONObject
+) : Parcelable, RawDataContainer<JSONObject> {
+    @IgnoredOnParcel
```

Comment:
nitpick, add an extra line above this one

Reply:
👍